python/lsst/obs/base/makeRawVisitInfoViaObsInfo.py

```diff
@@ -0,0 +1,172 @@
+# This file is part of obs_base.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import astropy.units
+from astropy.utils import iers
+
+# This is an unofficial ERFA interface
+import astropy._erfa as erfa
```

Comment:
This seems slightly worrying; mind augmenting the comment with a bit more on either why we shouldn't worry or why we have no choice?

Reply:
Yes, and no. If Astropy ever drops erfa then that will be a problem. On the other hand, the erfa routine in question is pretty simple to reproduce, and there is a python/erfa wrapper extant that we could use.